recipes/flang-activation/activate.sh

```diff
@@ -0,0 +1,2 @@
+export CONDA_BACKUP_FC=$FC
+export FC="flang-new"
```

Comment:
```suggestion
export FC="@CHOST@-flang"
```

Reply:
with @CHOST@ replaced

Reply:
Unix-only? Or also on windows?

Reply:
Unix-only. Windows is annoying due to no symlinks.

Reply:
On windows this isn't going to work so easily, since we don't have the symlinks.

Reply:
sorry, my tab didn't refresh to show your comments